app.py

```diff
@@ -1,48 +1,64 @@
 import requests
 import calendar
+from datetime import datetime
 import json
+from dotenv import load_dotenv
+import os
 
-print("type name of city and hit enter to get a 3 day forecast:")
-city = input()
+def configure():
+    load_dotenv()
 
 class Weather:
-    def __init__(self, city):
+    def __init__(self, city, days):
         self.city = city
+        self.days = days
     
     def forecast(self):
+        configure()
         url = "https://weatherapi-com.p.rapidapi.com/forecast.json"
 
-        querystring = {"q":f"{self.city}","days":"3"}
+        querystring = {"q":f"{self.city}","days":f"{self.days}"}
 
         headers = {
-            "X-RapidAPI-Key": "187b0ec4b7msh9909e74c3e40f95p17b4b6jsn67ee1ffae374",
+            "X-RapidAPI-Key": os.getenv('apikey'),
             "X-RapidAPI-Host": "weatherapi-com.p.rapidapi.com"
         }
 
         response = requests.get(url, headers=headers, params=querystring)
 
         data = response.json()
 
-        formatted = json.dumps(data,indent=2)
+        formatted_data = json.dumps(data,indent=2)
 
-        o = json.loads(formatted)
+        weatherObject = json.loads(formatted_data)
 
-        location = o["location"]
-        current = o["current"]
-        forecast = o["forecast"]['forecastday']
+        location = weatherObject["location"]
+        current = weatherObject["current"]
+        forecast = weatherObject["forecast"]['forecastday']
 
 
         print(f'It is currently {current["temp_f"]} degrees fahrenheit and {current["condition"]["text"]} in {location["name"]}, {location["country"]}')
 
-        count = 0
+        dt = datetime.now()
+        count = dt.weekday()
 
         for days in forecast:
             count += 1
+            if count > 6:
+                count -= 7
             day = days["day"]
             print(f'{calendar.day_name[count]} will have a high of {day["maxtemp_f"]}, and a low of {day["mintemp_f"]}, and will be {day["condition"]["text"]}')
 
-threeday = Weather(city)
-threeday.forecast()
+def main():
+    print("type name of city and hit enter:")
+    city = input()
+    print("Now please enter the number of days you would like a forecast for (please keep to 3 and under):")
+    days = input()
```

Comment:
Add a conditional statement here to validate the user input is within the number of days limit. Allow the user to enter the correct number of days if they entered > 3.